### PR TITLE
Implementing icon displaying and api styling

### PR DESCRIPTION
**Pull request by CalyCherkaoui**
In this PR, I implemented the styling of the API elements and the display of the weather icons

### CODE REVIEW

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it Git Flow you have a professional README file. 

However, in order to accomplish all requirements, you required to do these changes, listed below:
## General requirements 
- Edit pull request message following this [guideline](https://cdn.filestackcontent.com/osvFJEkQHWQsgDe0PJoQ)
- For future projects please try to add more descriptive commit messages explaining what and why made those changes follow this [tutorial](https://chris.beams.io/posts/git-commit/)  
- Install webpack-dev-server dependency, because you need to test the application. Run this `npm install webpack-dev-server --save-dev` statement
- Please make sure next time  you are making a pull request from a branch where you have the most of the changes because it's difficult o see on what changes you have worked
- It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
  - Please move all the methods that have function to arrow function. Follow this example 
image
### FILE: README.md
- Add the web page screenshot, to got to know what this application is
- Add the right live demo link 
### FILE: src/weather.js
- The `windDirection()` method, make `cadranNamesList` variable into a static one, and then you could use it, keep in mind that a function has to make only one thing. Follow this example
image
### FILE: src/index.js
- Fix the method name line 10 it should be `displayWeatherRequestForm`. Please from he original method 
- Fix the grammar speel to the next variables (switcher, submit, mainTempC)
- In this method `switcherTemp.addEventListener`, line 57. Refactor using only a ternary. Follow this example
image
### FILE: src/api.js
- Refactor this method `extractRawData()`. Line 9 you could return directly de `response.json()` without any wait statements because this method and tresponse.json() statement are already a promise. Follow this example
image

### OPTIONAL
* [ ] Add a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/loading_component_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
* [ ] You could add the current geolocation of the user when the app is first accessed and fetches the weather from that location
### Summary
-  Work using ES6 syntax, especially for arrow functions 
- The live demo link is missing
- Refactor some methods to have a clean code
***

Guys work on the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: README.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ This project project was built following the specifications , and user storiess
 ## Presentation/Demo:
 
 - Video-presentation of the project
```
Comment (addod19) [inline README.md]: Kindly change this to a live version of the project. There is no video presentation, the link when tested is the live version of the project, although the link does not show on the README

File: src/api.js [hunk 2]
```diff
@@ -5,10 +5,27 @@ const extractRawData = async (location) => {
     // fetch data from api
     const response = await fetch(apiUrl);
 
+    // parse promise data into json
+    // const data = await response.json();
```
Comment (addod19) [inline src/api.js]: Kindly remove commented code

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @CalyCherkaoui, great job on this project :+1: Having fixed most of the comments outlined by the previous TSE :+1: There are a few comments that need addressing. Kindly check the comments below:

- The command `npm run start` returns an error, so I tested the app with the `npm run build` command
- Kindly remove commented code

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: README.md (hunk 1 above)
Comment (CalyCherkaoui) [reply README.md]: Issue solved. Thank you for your feedback

File: src/api.js (hunk 2 above)
Comment (CalyCherkaoui) [reply src/api.js]: Issue solved. Thank you for your feedback

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x]  i still can't do `npm run start`, i fixed it by changing  `package.json`, on line 9, to 
```json
"start": "webpack serve --mode development --open",
```

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**